src/test/CMakeLists.txt

```diff
@@ -607,6 +608,26 @@ set_property(TEST
   WITH_PYTHON2=${WITH_PYTHON2}
   WITH_PYTHON3=${WITH_PYTHON3}
   )
+else()
```

Comment:
i don't really like to repeat these settings, especially when they are lengthy.
how about this:
```cmake
set(tox_tests run-tox-ceph-disk run-tox-ceph-detect-init)
set(env_vars_for_tox_tests
  CEPH_BUILD_DIR=${CMAKE_BINARY_DIR}
  CEPH_ROOT=${CMAKE_SOURCE_DIR}
  CEPH_BIN=${CMAKE_RUNTIME_OUTPUT_DIRECTORY}
  CEPH_LIB=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
  CEPH_BUILD_VIRTUALENV=${CEPH_BUILD_VIRTUALENV}
  CEPH_DISK_VIRTUALENV=${CEPH_DISK_VIRTUALENV}
  CEPH_DETECT_INIT_VIRTUALENV=${CEPH_DETECT_INIT_VIRTUALENV}
  LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/lib
  PATH=$ENV{PATH}:${CMAKE_RUNTIME_OUTPUT_DIRECTORY}:${CMAKE_SOURCE_DIR}/src
  PYTHONPATH=${CMAKE_SOURCE_DIR}/src/pybind
  WITH_PYTHON2=${WITH_PYTHON2}
  WITH_PYTHON3=${WITH_PYTHON3})
if(WITH_MGR)
  list(APPEND tox_tests run-tox-mgr-dashboard)
  set(MGR_DASHBOARD_VIRTUALENV ${CEPH_BUILD_VIRTUALENV}/mgr-dashboard-virtualenv)
  list(APPEND env_vars_for_tox_tests MGR_DASHBOARD_VIRTUALENV=${MGR_DASHBOARD_VIRTUALENV})
endif()
set_property(
  TEST ${tox_tests}
  PROPERTY ENVIRONMENT ${env_vars_for_tox_tests})
```